src/srsnv/ugbio_srsnv/reports/srsnv_report.ipynb

```diff
@@ -734,6 +750,36 @@
     "The number in brackets next to categorical features refers to the appropriate colormap on the right that corresponds to the possible values of the feature. "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "4946fbee",
+   "metadata": {},
+   "source": [
+    "# Glossary of terms\n",
+    "\n",
+    "- **ML_qual** is the output of the SRSNV model, in phred scale: $$\\mathrm{ML\\_qual} = -10 \\log_{10}(1-\\hat{p}),$$ where $\\hat{p}$ is the model prediction (estimate of the probability that the SNV is not noise).\n",
+    "\n",
+    "- **Filter Quality (FQ)** quantifies the reliability of a filtering strategy for SNVs. Given a filter that selects a subset of SNVs, the FQ is defined as:  $$\\text{FQ} = -10 \\log_{10} \\left( \\frac{\\text{FPR}}{\\text{TPR}} \\right)$$ where **FPR** (false positive rate) and **TPR** (true positive rate) are measured among the SNVs that pass the filter. See [Filter Quality (FQ) vs Recall](#Filter-Quality-%28FQ%29-vs-Recall) for a more precise description of how FQ is estimated.  \n",
+    "The FQ score is related to the precision of the filter through the **base rate** $b$, the fraction of all reads that intersect loci with a true SNV:\n",
+    "$1 - \\text{Precision} \\approx \\frac{\\text{FPR}}{b\\,\\text{TPR}}$, where the approximation holds when $\\frac{\\text{FPR}}{\\text{TPR}} \\ll b \\ll 1$.\n",
+    "Expressed in phred scale: $\\text{phred}(1 - \\text{Precision}) \\approx \\text{FQ} + 10 \\log_{10}(b)$.\n",
+    "\n",
+    "- **SNVQ**. The SRSNV model is used to filter SNVs by retaining only those with ML_qual values above a certain threshold. This filtering startegy defines a calibration function that maps the threshold ML_qual value to the FQ of the resulting filter (see [ML_qual -> SNVQ mapping function](#ML_qual--%3E-SNVQ-mapping-function)). SNVQ is then defined to be this calibration function applied to the ML_qual of the SNV. In other words, it is the FQ of the filter whose threshold is the SNV's ML_qual.\n",
+    "\n",
+    "\n",
+    "<h3>ppmSeq tags</h3>  \n",
+    "\n",
+    "Except where otherwise stated, \n",
+    "- **mixed** or **mixed, both ends** refers to reads where both read ends have MIXED ppmSeq tags. \n",
+    "- **mixed, start** refers to reads where the start read ppmSeq tag is MIXED, regardless of the end tag. \n",
+    "Note that **mixed** reads are included in **mixed, start** reads. See [SNVQ vs start/end ppmSeq tag](#SNVQ-vs-start/end-ppmSeq-tag) for more detailed information about ppmSeq tags. \n",
+    "\n",
+    "<h3>Training set lables</h3>  \n",
+    "\n",
+    "- **TP** refers to reads with a homozygous SNV. They are labeled as *True*.\n",
+    "- **FP** refers to reads with SNVs that are supported by a single read. They are labeled as *False*."
```

Comment:
Should be:
- **TP** refers to homozygous SNV. They are labeled as True.
- **FP** refers to SNVs that are supported by a single read (in a locus with a high coverage). They are labeled as False.